kse/src/org/kse/gui/dialogs/resources.properties

```diff
@@ -561,3 +561,8 @@ RevokedCertsTableHeadRend.SerialNumberColumn.tooltip   = Serial number of revoke
 
 RevokedCertsTableModel.RevocationDateColumn = Revocation Date
 RevokedCertsTableModel.SerialNumberColumn   = Serial Number
+
+DCompareCertificates.Title          = Compare Certificates
+DCompareCertificates.jbOK.text      = Ok
+DCompareCertificates.KeyUsage       = Key Usage: {0}
+DCompareCertificates.jlMatch.text   = % of matching: {0}
```

Comment:
Here is the French version, for:
- kse/src/org/kse/gui/dialogs/resources_fr.properties

```ini
DCompareCertificates.Title          = Comparer les certificats
DCompareCertificates.jbOK.text      = Valider
DCompareCertificates.KeyUsage       = Usage de clef (KU)\u00A0: {0}
DCompareCertificates.jlMatch.text   = Correspondance\u00A0: {0}\u00A0%
```

- Could you update the French resource? or Could you need a PR from me?

Regards.

Reply:
added thanks.